### ISSUE TITLE
Error while loading Zeus Android with MTE enabled (memory tagging)

### ISSUE THREAD

**Issue by bedair81** [ZeusLN/zeus#1872]
### Describe the bug

enabling MTE to detect unsafe memory code on grapheneOS causes zeus wallet to terminate while starting the node on android 14 zeus 0.8.0 beta 4 

### Reproduce

enable MTE, start zeus, startup terminated

### ZEUS version

v0.8.0

### Node interface

Embedded LND

### Network

Clearnet

### Device

pixel 8 pro

### Device operating system

android 14 GOS

### Log output

```shell
type: crash
osVersion: google/husky/husky:14/UD1A.231105.004/2023112600:user/release-keys
uid: 10150 (u:r:untrusted_app_32:s0:c150,c256,c512,c768)
cmdline: app.zeusln.zeus
processUptime: 14s

signal: 11 (SIGSEGV), code 9 (SEGV_MTESERR), faultAddr 400cce9285b8330
threadName: Thread-4
MTE: enabled

backtrace:
    /data/app/~~vqt6U_EiZVbjVoRUaeFX1Q==/app.zeusln.zeus-BmWtxRhhpao29yIiWNKqPQ==/lib/arm64/libgojni.so (pc 456f34)
```

**Comment by bedair81** [comment]
after updating the 0.8.0 beta 5, the same crash with MTE enabled occurs

**Comment by bedair81** [comment]
Issue has been reported to React Native at: https://github.com/facebook/react-native/issues/44276

**Comment by bedair81** [comment]
Issue seems to stem from native "Go" code running in embedded LND, doesn't occur if booting app and loading a remote node for management. Unsure whether to report the issue to "Go" themselves or LND

**Comment by bedair81** [comment]
This seems to be resolved on 0.9.0 beta 3, not sure what fixed it but this is great news!
